tensorflow/compiler/jit/kernels/xla_ops.h

```diff
@@ -175,6 +175,16 @@ class XlaRunOp : public OpKernel {
   const XlaPlatformInfo platform_info_;
 };
 
+class XlaMergeOp : public OpKernel {
```

Comment:
Can you just use `tensorflow::MergeOp` instead of creating a new implementation?

Reply:
I initially tried to use the same implementation. The allocation for the value_index scalar (through context->allocate_output(...) doesn't work when called through XLA. I then tried an implementation using GetAllocator(..., patform_info_) which did work. I subsequently decided to limit the _XlaMerge op to not even have a value index. I didn't see a need for it, and build_xla_ops_pass.cc doesn't use it either.
(When removing the index, I forgot to remove platform_info_, which you commented about earlier).

Reply:
Ok.  Please add a comment on the `.Doc` for `_XlaMerge` that it intentionally does not have the `value_index` output.

Reply:
done